bitnami/opensearch/templates/_helpers.tpl

```diff
@@ -0,0 +1,679 @@
+{{- end -}}
+
+{{/*
+Get the initialization scripts Secret name.
+*/}}
+{{- define "opensearch.initScriptsSecret" -}}
+{{- printf "%s" .Values.initScriptsSecret -}}
+{{- end -}}
+
+{{/*
+ Create the name of the master service account to use
+ */}}
+{{- define "opensearch.master.serviceAccountName" -}}
+{{- if .Values.master.serviceAccount.create -}}
+    {{ default (include "opensearch.master.fullname" .) .Values.master.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.master.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+ Create the name of the coordinating-only service account to use
+ */}}
+{{- define "opensearch.coordinating.serviceAccountName" -}}
+{{- if .Values.coordinating.serviceAccount.create -}}
+    {{ default (include "opensearch.coordinating.fullname" .) .Values.coordinating.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.coordinating.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+ Create the name of the data service account to use
+ */}}
+{{- define "opensearch.data.serviceAccountName" -}}
+{{- if .Values.data.serviceAccount.create -}}
+    {{ default (include "opensearch.data.fullname" .) .Values.data.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.data.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+ Create the name of the ingest service account to use
+ */}}
+{{- define "opensearch.ingest.serviceAccountName" -}}
+{{- if .Values.ingest.serviceAccount.create -}}
+    {{ default (include "opensearch.ingest.fullname" .) .Values.ingest.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.ingest.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the opensearch TLS credentials secret for master nodes.
+*/}}
+{{- define "opensearch.master.tlsSecretName" -}}
+{{- $secretName := .Values.security.tls.master.existingSecret -}}
+{{- if $secretName -}}
+    {{- printf "%s" (tpl $secretName $) -}}
+{{- else -}}
+    {{- printf "%s-crt" (include "opensearch.master.fullname" .) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the opensearch TLS credentials secret for data nodes.
+*/}}
+{{- define "opensearch.data.tlsSecretName" -}}
+{{- $secretName := .Values.security.tls.data.existingSecret -}}
+{{- if $secretName -}}
+    {{- printf "%s" (tpl $secretName $) -}}
+{{- else -}}
+    {{- printf "%s-crt" (include "opensearch.data.fullname" .) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the opensearch TLS credentials secret for ingest nodes.
+*/}}
+{{- define "opensearch.ingest.tlsSecretName" -}}
+{{- $secretName := .Values.security.tls.ingest.existingSecret -}}
+{{- if $secretName -}}
+    {{- printf "%s" (tpl $secretName $) -}}
+{{- else -}}
+    {{- printf "%s-crt" (include "opensearch.ingest.fullname" .) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the opensearch TLS credentials secret for coordinating-only nodes.
+*/}}
+{{- define "opensearch.coordinating.tlsSecretName" -}}
```

Comment:
This is not being used

Reply:
[a2fb893](https://github.com/bitnami/charts/pull/17509/commits/a2fb893cda02e4ba56a3234adbb9569395c081c8)